modules/serverless-workflow-autoscaling-kn.adoc

```diff
@@ -0,0 +1,28 @@
+// Module is included in the following assemblies:
+//
+// serverless/knative_serving/configuring-knative-serving-autoscaling.adoc
+
+[id="autoscaling-workflow-kn_{context}"]
+= Autoscaling workflow using the Knative `kn` CLI
+
+You can edit autoscaling capabilities for your cluster by using `kn` to modify Knative services without editing YAML files directly.
+
+You can use the `kn service create` and `kn service update` commands with the appropriate flags as described below to configure autoscaling behavior.
+
+[cols="1,3",options="header"]
+|====
```

Comment:
While `====` (4) does work as-is, technically for tables it only needs to be `===` (3) here, and in the closing block.